core/arch/x86/decode_fast.c

```diff
@@ -1353,13 +1352,15 @@ decode_cti(dcontext_t *dcontext, byte *pc, instr_t *instr)
                 byte1 = *(pc + prefixes);
                 break;
             case EVEX_PREFIX_OPCODE:
+                instr_set_prefix_flag(instr, PREFIX_EVEX);
```

Comment:
Please ensure this is set in decode() as well.  I would expect the test to fail if not I suppose since you have a bb event which should force full_decode.

Reply:
Yes it is set through the evex flag in the decode tables. The test would have failed, yes.